src/test/java/com/google/sps/ContextServletTest.java

```diff
@@ -0,0 +1,60 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package com.google.sps;
+
+import static org.mockito.Mockito.atLeast;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+
+import com.google.sps.servlets.ContextServlet;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import java.security.GeneralSecurityException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+/** This is a JUnit test for sentiment mockedAnalysis */
+@RunWith(JUnit4.class)
+public class ContextServletTest {
+  private static final String URL_PARAMETER = "url";
+  private static final String TEST_URL = "WkZ5e94QnWk";
+  private static final String REQUEST_INFO = "snippet,contentDetails,statistics";
+
+  @Test
+  public void testContextGet() throws ServletException, IOException, GeneralSecurityException {
+    ContextServlet contextServlet = new ContextServlet();
```

Comment:
We'll likely want to change this test to mock out the dependency on the Youtube API so that we don't actually need to send an external request in the test.

Primarily, the goal of the test should be to ensure that we call the Youtube API w/ the arguments that we expect + that the JSON that we return is constructed as we expect given a specific response from the Youtube API.